### DISCUSSION TITLE
Is it possible to copy clipping path from 8BIM from one image to an another image

### DISCUSSION THREAD

**Discussion by fire1** [ImageMagick/ImageMagick#3557] [Help]
Is it possible to copy 8mib's only clipping path SVG from one image to another using convert?
This is the command to export clipping paths

`identify -quiet -format "%[8BIM:1999,2998:#1]" clipping-paths.jpg`

I read most of the documentation of ImageMagick and all I found is an image editing, no image data export/import 
(or didn't get how it will work...)

**Comment by snibgo** [comment]
What version of IM, on what platform?

If clipping-paths.jpg contain a clipping path, then this command:
```
magick clipping-paths.jpg out.jpg
```
... will result in out.jpg also containing the clipping path.

**Comment by snibgo** [comment]
"8BIM" is a profile. We can save this profile as a binary file, then assign it to another image. For example, we save the clipping path into path.bin, then assign that profile to toes.png, saving the result as out.png.
```
magick clipping-paths.jpg +write 8BIM:path.bin +delete toes.png -profile path.bin out.png
```
out.png then has the pixels from toes.png, and the clipping path from clipping-paths.jpg.

Does that help? If not, please say what you are trying to do.

**Reply by fire1** [reply]
The platform is ImageMagick 7.0.8-68 Q16 x86_64 /  Ubuntu

Thanks for the help, but in my case, it is required not to "touch" the original except clipping paths.
I'm parsing JPG, TIF,  and in some cases PSDs that contain different meta-data Infos and color profiles with layers. 
I am currently transferring only 8bim profiles via PHP, but in some images, the color profile is destroyed or some layers are changed. I guess is caused by some bad re-arrangement of binary data from PHP (with IM7.0.11 TIFs was a huge nightmare )

I will try to make a workaround with your approach.
P.S. The final "boss" will be to parse from PHP the binary of the 8mib profile like how is made in IM  and transfer it like this way.
